app/src/components/Title.tsx

```diff
@@ -0,0 +1,11 @@
+export default function Title({ text }: { text?: string }) {
```

Comment:
Consider making the text mandatory, we would not want to miss one and have a "DEFAULT TITLE" somewhere
```suggestion
export default function Title({ text }: { text: string }) {
```